components/backstage/components/backstage_section.tsx

```diff
@@ -1,35 +1,30 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import PropTypes from 'prop-types';
 import React from 'react';
 import {NavLink} from 'react-router-dom';
 
-export default class BackstageSection extends React.PureComponent {
-    static get propTypes() {
-        return {
-            name: PropTypes.string.isRequired,
-            title: PropTypes.node.isRequired,
-            parentLink: PropTypes.string,
-            subsection: PropTypes.bool,
-            children: PropTypes.arrayOf(PropTypes.element),
-            id: PropTypes.string,
-        };
-    }
+type Props = {
+    name: string;
+    title: React.ReactNode;
+    parentLink?: string;
+    subsection?: boolean;
+    children: React.ReactElement[];
+    id?: string;
```

Comment:
should id be optional? I see we didn't require it originally but it looks to me that it should be required